docs/source/config/users.md

```diff
@@ -14,3 +14,12 @@ const options = {
   }
 };
 ```
+### Storing additional values
+
+The standard user data type stores most of the information a user could have. But if you want to store additional information in user, you can do so by using the `meta` field on  `UserProfileInput` that is passed to the `Mutation.updateUserProfile`:
```

Comment:
Now that meta is on root, it does not make sense to provide meta in UserProfileInput, it should be another toplevel:

The standard user data type stores most of the information a user could have. But if you want to store additional information in user, you can do so by using the `meta` field ~~on  `UserProfileInput`~~ that is passed to the `Mutation.updateUserProfile`: 

->

The standard user data type stores most of the information a user could have. But if you want to store additional information in user, you can do so by using the `meta` field as a property on `Mutation.updateUserProfile`